### ISSUE TITLE
Refactor application_controller.rb methods

### ISSUE THREAD

**Issue by scott** [helpyio/helpy#789]
The following methods in `application_controller` raise an error if certain features are disabled:

`knowledgbase_enabled?`
`forums_enabled?`

Instead of raising, they should redirect or return false.  We have seen with production instances, some spiders blindly trying URLs and this results in unneeded exceptions.

**Comment by CGA1123** [comment]
The `raise ActionController::RoutingError.new('Not Found')` does cause the web interface to show a 404 page.

Which is the same as what happens when a request to a random page is done as well (eg `/en/lalallalalalal`).

**Comment by scott** [comment]
Yeah but it also causes an exception message written to the logs/appsignal/rollbar etc, which is what I am thinking could be reduced...
